### PR TITLE
Refactor/video playback and content events

### PR DESCRIPTION
**Pull request by brennan**
- Refactor case for "Video Playback" events
- Add case for "Video Playback" events
- Update relevant tests

### CODE REVIEW

File: src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java [hunk 1]
```diff
@@ -404,13 +403,12 @@ private void trackVideo(String eventName, TrackPayload track) {
...
       case "Video Playback Started":
-        Context context = analytics.getApplication();
-        Properties properties = track.properties();
+        Properties videoPlaybackProperties = track.properties();
         MediaHeartbeatConfig config = new MediaHeartbeatConfig();
 
         config.trackingServer = heartbeatTrackingServer;
-        if (properties.get("channel") != null) {
-          config.channel = properties.getString("channel");
+        if (videoPlaybackProperties.get("channel") != null) {
```
Comment (f2prateek) [inline src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java]: These !=null checks should be refactored into a helper method.
```
String getDefault(ValueMap map, String key, String defaultValue) {
String value = map.getString(key);
if (value == null) return defaultValue;
return value;
}
```

File: src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java [hunk 2]
```diff
@@ -404,13 +403,12 @@ private void trackVideo(String eventName, TrackPayload track) {
 
     switch (eventName) {
       case "Video Playback Started":
```
Comment (f2prateek) [inline src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java]: Some of these cases are pretty big. These should be refactored into their own functions (e.g. this should be `VideoPlaybackStarted`.

**Comment by brennan** [comment]
@f2prateek I don't think it makes sense to break Video Playback Started refactor into a separate PR since the change is only three lines. I thought about this before pushing. Anyway, we're already crunched for time, so that would eat into more productivity.

**Comment by f2prateek** [comment]
@brennan I think this should absolutely be two PRs. Big PRs are hard to review and looking at multiple things at once makes it harder to spot bugs.

**Comment by f2prateek** [comment]
If we're strapped for time, feel free to not refactor "Video Playback Started" at all and just do "Add Video Content Started"  as it's own PR.

**Pull request closed without merging**